pkg/reconciler/ingress/resources/route_test.go

```diff
@@ -0,0 +1,341 @@
+package resources
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	routev1 "github.com/openshift/api/route/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"knative.dev/serving/pkg/apis/networking"
+	networkingv1alpha1 "knative.dev/serving/pkg/apis/networking/v1alpha1"
+	"knative.dev/serving/pkg/apis/serving"
+)
+
+const (
+	localDomain     = "test.default.svc.cluster.local"
+	externalDomain  = "public.default.domainName"
+	externalDomain2 = "another.public.default.domainName"
+
+	lbService   = "lb-service"
+	lbNamespace = "lb-namespace"
+
+	uid       = "8a7e9a9d-fbc6-11e9-a88e-0261aff8d6d8"
+	routeBase = "route-" + uid
+)
+
+func TestMakeRoutes(t *testing.T) {
+	tests := []struct {
+		name    string
+		ingress *networkingv1alpha1.Ingress
+		want    []*routev1.Route
+		wantErr error
+	}{
+		{
```

Comment:
Let's collapse these a bit. You can throw `}{{` on one line, `}, {` on one line between the cases and then end it with `}}` to get rid of some indentation.